extensions/jqt.menusheet.js

```diff
@@ -149,4 +203,4 @@
     } else {
         console.error('Extension `jqt.menusheet` failed to load. jQT not found');
     }
-})($);
+}( jQuery || $ )); // jQuery or jQuery-like library, such as Zepto
```

Comment:
@illucent, do we really need to change it? I think we have test that passing on both.